field.py

```diff
@@ -0,0 +1,123 @@
+import random
+from random import randint
+import cell
+import bear
+import constants
+
+list_of_bears = []
+
+out_file = open('bear_out.txt', 'w')
+
+def create_bear(name, cell_row, cell_col):
+    activity_level = random.random()
+    smell = random.random()
+    aggression = random.random()
+    cowardice = random.random()
+    life_count = constants.INITIAL_LIFE_LEVEL
+
+    return bear.Bear(
+        name,
+        activity_level,
+        smell,
+        aggression,
+        cowardice,
+        life_count,
+        cell_row,
+        cell_col)
+
+
+def populate_field(field_instance):
+
+    honey_counter = constants.HONEY_AMOUNT
+
+    while honey_counter > 0:
+        row = random.randint(0, constants.MAX_ROW - 1)
+        col = random.randint(0, constants.MAX_COL - 1)
+
+        if not field_instance.get_cell_at_location(row, col).has_honey:
+            field_instance.set_honey(row, col)
+            honey_counter = honey_counter - 1
+
+    bear_counter = constants.BEARS_AMOUNT
+
+    while bear_counter > 0:
+        random_empty_cell = field_instance.get_random_empty_cell()
+        new_bear = create_bear(str(bear_counter), random_empty_cell.row, random_empty_cell.col)
+        list_of_bears.append(new_bear)
+        random_empty_cell.set_bear(new_bear)
+        bear_counter = bear_counter - 1
+
+
+class Field(object):
+
+    def __init__(self):
+        """
+            Args:
+        """
+        self.matrix = []
```

Comment:
Redundant line, value gets overwritten in the next line

Reply:
Don't I need to initialize the variable in this way?

Reply:
No, the line in the next row sets a new list value

Reply:
fixed